west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 0a3f2f0397a86425cc7d12fa3a0c0ab8020d80e1
+      revision: 4d0a8c1e8b8316ec5071a3fcc98cd292d3bc4e6d
```

Comment:
I checked, and if you checkout one commit earlier, PTL doesn't build. This commit probably should be the first.

Reply:
@pjdobrowolski see my comment in https://github.com/thesofproject/sof/pull/9220#pullrequestreview-2127832852 .. when adding new capabilities to SOF, it's recommended to do the Zephyr update in a separate PR, and then do a SOF pr to use the new capabilities.

Having the zephyr update in this series will make it more complicate to get this PR through as you'll get merge conflicts when the baseline is updated (for other reasons). As long as we don't have PTL in the CI, you can't really any break any build target yet.

Reply:
Removed and moved to new PR https://github.com/thesofproject/sof/pull/9241